src/index.html

```diff
@@ -27,13 +27,10 @@
 
   </head>
 
-  <body ng-strict-di>
-    <div ng-controller="dimAppCtrl as app" ng-class="{ 'show-elements': app.settings.showElements, itemQuality: app.featureFlags.qualityEnabled && app.settings.itemQuality, 'show-new-items': app.settings.showNewItems, 'new-item-animated': app.settings.showNewAnimation }">
-      <div class="ng-cloak" ng-click="app.trackActivity()">
-        <div ui-view></div>
-      </div>
-    </div>
-
+  <body ng-app="app" ng-csp ng-strict-di ng-cloak>
+    <app>
+      Loading...
```

Comment:
Angular hasn't loaded at this point so I don't have a means to apply i18n.  We can replace this with some animated graphic.  Spin DIM?  @chrisfried would finally have his baby see the light of day.